### ISSUE TITLE
[Feature Request] Preserve "Line Breaks"

### ISSUE THREAD

**Issue by barbedknot** [David-Byrne/Hangons#14]
>         {
>           "conversation_id": {
>             "id": "[censored]"
>           },
>           "sender_id": {
>             "gaia_id": "[censored]",
>             "chat_id": "[censored]"
>           },
>           "timestamp": "[censored]",
>           "self_event_state": {
>             "user_id": {
>               "gaia_id": "[censored]",
>               "chat_id": "[censored]"
>             },
>             "client_generated_id": "[censored]"
>           },
>           "chat_message": {
>             "message_content": {
>               "segment": [
>                 {
>                   "type": "TEXT",
>                   "text": "[censored]",
>                   "formatting": {
>                     "bold": false,
>                     "italics": false,
>                     "strikethrough": false,
>                     "underline": false
>                   }
>                 },
>                 {
>                   "type": "LINE_BREAK",
>                   "text": "\n"
>                 },
>                 {
>                   "type": "LINE_BREAK",
>                   "text": "\n"
>                 },
>                 {
>                   "type": "TEXT",
>                   "text": "[censored]",
>                   "formatting": {
>                     "bold": false,
>                     "italics": false,
>                     "strikethrough": false,
>                     "underline": false
>                   }
>                 }
>               ]
>             }
>           },

The above is an excerpt of a "Hangouts.json" to provide an example of a hangouts message that uses "line breaks", which can be done without copy-pasting by holding shift + enter. This example message pressed shift + enter twice, to separate two paragraphs. This formatting isn't currently preserved in any of the output formats "Hangons" offers. Is there any chance you will ever add support for this for any output method? Thanks for the consideration.

Also, the google docs form provided on the project page for feedback isn't working, it says "Sorry, the file you have requested has been deleted."

**Comment by David-Byrne** [comment]
Yeah that sounds like a good feature. I don't have capacity to implement it but I'll happily review any PRs for it
